.env.local

```diff
@@ -1 +1 @@
-NEXT_PUBLIC_REST_URL=http://localhost:7001
+NEXT_PUBLIC_REST_URL=http://localhost:7002
```

Comment:
ig this change was not intended

Reply:
Yeah! I think we put this inside the untrack

Reply:
yup that would work